packages/gatsby-remark-code-repls/src/gatsby-node.js

```diff
@@ -36,45 +36,46 @@ exports.createPages = (
     )
   }
 
-  // TODO We could refactor this to use 'recursive-readdir' instead,
-  // And wrap with Promise.all() to execute createPage() in parallel.
-  // I'd need to find a way to reliably test error handling though.
-  const files = recursiveReaddir(directory)
+  try {
```

Comment:
just wondering why do a try catch if you're going to rethrow anyway?

Reply:
That’s true, thanks. I’m wondering what is better between omitting the `try / catch` altogether or adding a simple `console.error` in the `catch` with a (hopefully) helpful error message, e.g. 

> Error in gatsby-remark-code-repls plugin: cannot read directory ${directory}

What do you think @wardpeet

Reply:
I missed this one 🤦‍♂️ reporter.panic is probably a great way! Especially if you can give a good error message like.

> Oh this went wrong, please check ...

Reply:
@wardpeet Didn’t know about `reporter.panic`, neat! I updated the PR to use it and enhance the error handling. Thanks!